### PR TITLE
Feature/viewer-context-detection

### PR DESCRIPTION
**Pull request by ZacZhangzhuo**
### What type of change is this?

- [x] Added `viewerinstance ` in  `compas.scene.Scene` to support viewers context detection.
- [x] Some import sorting.

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [x] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [x] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [x] I have added necessary documentation (if appropriate)

In the viewer, checks can be run like:

```python
from compas.scene.context import _detect_current_context
from compas_viewer import Viewer

viewer = Viewer()
print(_detect_current_context())
```

This PR is associtaed to https://github.com/compas-dev/compas_viewer/issues/12 @tomvanmele @Licini

### CODE REVIEW

**Comment by Licini** [comment]
@ZacZhangzhuo linter error

**Comment by ZacZhangzhuo** [comment]
This one should be great @Licini

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
-
+* Added `viewerinstance ` in  `compas.scene.Scene` to support viewers context detection.
```
Comment (tomvanmele) [inline CHANGELOG.md]: there are some superfluous spaces here

Comment (ZacZhangzhuo) [reply CHANGELOG.md]: done

**Review by Licini — APPROVED** [review]

**Pull request merged**